e2e/test.ts

```diff
@@ -137,26 +121,32 @@ async function runCallProtectedScenario(
 async function runTest() {
   // Show help if requested
   if (args.includes('-h') || args.includes('--help')) {
-    console.log('🚀 X402 E2E Test Suite');
-    console.log('======================');
-    console.log('');
     console.log('Usage: npm test [options]');
     console.log('');
     console.log('Options:');
-    console.log('  -v [test_numbers...]       Enable verbose logging for specific test numbers');
+    console.log('Environment:');
+    console.log('  -d, --dev                  Development mode (base-sepolia, no CDP)');
+    console.log('  -v, --verbose              Enable verbose logging');
+    console.log('  -ts, --typescript          Include TypeScript implementations');
+    console.log('  -py, --python              Include Python implementations');
+    console.log('  -go, --go                  Include Go implementations');
+    console.log('');
+    console.log('Filters:');
     console.log('  --log-file=<path>          Save verbose output to file');
     console.log('  --client=<name>            Filter by client name (e.g., httpx, axios)');
     console.log('  --server=<name>            Filter by server name (e.g., express, fastapi)');
-    console.log('  --network=<name>           Filter by network (e.g., base, base-sepolia)');
-    console.log('  --facilitator=<true|false> Filter by facilitator usage');
+    console.log('  --network=<name>           Filter by network (base, base-sepolia)');
+    console.log('  --prod=<true|false>        Filter by production vs testnet scenarios');
+    console.log('  --language=<name>          Filter by language (e.g., typescript, python, go)');
     console.log('  -h, --help                 Show this help message');
     console.log('');
     console.log('Examples:');
-    console.log('  npm test -- -v              # Run all tests with verbose logging');
-    console.log('  npm test -- -v 54           # Run all tests, verbose for test #54');
-    console.log('  npm test -- -v 54 55 56     # Run all tests, verbose for tests #54, #55, #56');
-    console.log('  npm test -- -v --log-file=test.log  # Save all verbose output to test.log');
-    console.log('  npm test -- --client=httpx --server=express -v 1');
+    console.log('  pnpm test                         # Run all tests');
+    console.log('  pnpm test -d                      # Run tests in development mode');
+    console.log('  pnpm test -py -go                 # Test Python and Go implementations');
+    console.log('  pnpm test -ts --client=axios      # Test TypeScript axios client');
+    console.log('  pnpm test -d -py                  # Dev mode, Python implementations only');
+    console.log('  pnpm test --network=base --prod=true');
```

Comment:
```suggestion
    console.log('  pnpm test --network=base --prod=true');  # Base mainnet
```

(or something like that)

Reply:
I was struggling with that to call this, but `prod` here refers to what facilitator to filter down to.

By default, no filter, so you get every facilitator, cdp and x402.org
`--prod=true` means use the CDP facilitator only
`--prod=false` means filter down to x402.org only